### ISSUE TITLE
Improve flicker/lag when UI first appears

### ISSUE THREAD

**Issue by olilarkin** [iPlug2/iPlug2#26]
Since enabling transient IGraphics contexts ( so that no textures et cetera are loaded onto the GPU  until plug-in user interface is  required), there is a noticeable lag seeing the UI,  at least with virtualCZ.  this wasn't an issue when the metal context was created before the UI was needed

**Comment by johannesmenzel** [comment]
Yes, using NanoVG. When back on the code I'll try some things to figure out if it's a problem with my controls. But as far as I remember I had these issues from the beginning (means from duplicated IPlugEffect).

**Comment by johannesmenzel** [comment]
The severe lag of multiple seconds seems to be related to user specific errors (me) in my custom IControls.
